### PR TITLE
Devel Transport

### PR DESCRIPTION
**Pull request by bremoran**
Update sockets tests to new version of mbed-greentea, unity, and utest.

### CODE REVIEW

**Comment by bremoran** [comment]
cc @bogdanm @mazimkhan @0xc0170 @adbridge @niklas-arm

File: module.json [hunk 1]
```diff
@@ -21,7 +21,9 @@
     "minar": "^1.0.0"
   },
   "testDependencies": {
-    "mbed-drivers": "^1.0.0"
+    "mbed-drivers": "^1.0.0",
+    "unity": "^2.0.1",
+    "greentea-client": "^0.1.4"
```
Comment (bogdanm) [inline module.json]: "^0.1.4" means "==0.1.4" for major version 0, see the "dependencies" section in http://yottadocs.mbed.com/reference/module.html. Probably not the intention here.

**Comment by bogdanm** [comment]
Look good overall with the exception of the dependency spec issue.

**Comment by PrzemekWirkus** [comment]
@adbridge More details about changes to Greentea stuff here: https://github.com/ARMmbed/greentea/pull/78

**Pull request merged**